src/server/scripts/ScriptLoader.cpp

```diff
@@ -812,7 +812,7 @@ void AddEasternKingdomsScripts()
     AddSC_undercity();
     AddSC_western_plaguelands();
     AddSC_westfall();
-    AddSC_wetlands();
+    //AddSC_wetlands();
```

Comment:
why did you remove wetlands?

Reply:
Because everything in that file has been converted to SAI.  It was only the Slim and Mikhail NPCs scripts.  

```SQL
void AddSC_wetlands()
{
    new npc_tapoke_slim_jahn();
    new npc_mikhail();
}
```
I just commented it out because I noticed that is how Alterac Mountains was done above it. 
Do you want me to do it a different way?  Should we just clear out that wetlands file?  I left it untouched.